apps/worker/src/app/workflow/usecases/add-job/add-job.usecase.ts

```diff
@@ -580,7 +633,14 @@ export class AddJob {
           status: ExecutionDetailsStatusEnum.PENDING,
           isTest: false,
           isRetry: false,
-          raw: JSON.stringify({ delay }),
+          raw: JSON.stringify({
+            delay,
+            ...(untilDate && {
```

Comment:
useful information in the activity feed to actually know for when the digest/delay is scheduled